Makefile

```diff
@@ -119,6 +121,12 @@ BUILDFLAGS += -ldflags "-X $(ROOT_PACKAGE)/pkg/features.FeatureFlagToken=$(FEATU
 RELEASE_ORG_REPO := cloudbees/cloudbees-jenkins-x-distro
 endif
 
+ifdef JUNIT_REPORT
+# REPORT_PIPE must go at end of line
+OUTFILE := $(shell mktemp)
+REPORT_PIPE := -test.v 2>&1 | tee $(OUTFILE) | go-junit-report > build/integration.junit.xml & tail -f $(OUTFILE)
```

Comment:
Should this be  $(REPORTS_DIR)/integration.junit.xml or do you see REPORTS_DIR only as target for html report?

Reply:
Where `go-junit-report` coming from? Should that not be added to `get-test-deps`?

Reply:
> Should this be $(REPORTS_DIR)/integration.junit.xml or do you see REPORTS_DIR only as target for html report?

Typo. Should be fixed now.

Reply:
> Where go-junit-report coming from? Should that not be added to get-test-deps?

The builder image. No.